armi/reactor/converters/uniformMesh.py

```diff
@@ -244,13 +236,33 @@ def _buildAllUniformAssemblies(self):
             src = sourceAssem.spatialLocator
             newLoc = self.convReactor.core.spatialGrid[src.i, src.j, 0]
             self.convReactor.core.add(newAssem, newLoc)
+        self.plotConvertedReactor()
 
     def plotConvertedReactor(self):
```

Comment:
@ntouran  What do you think of having a plotting method in a place like this?

I think it seems strangely out of place, and means we need to import matplotlib when we are just importing a reactor or converter.  I don't know... should we move all plotting tools to armi/utils/plotting.py, or similar?  Or am I crazy?

Reply:
This is just for debugging now and testing.

Reply:
Hmmm... I don't like things that are used for testing cluttering up our code though. This may have to get moved out of the class.